### PR TITLE
Fix form submission on enter key for Patient Details

### LINKED ISSUE #3727: Pressing enter from focused elements in Patient Registration opens up import external results page.
### Steps to reproduce:
- 1. Edit a patient detail
- 2. Press enter after focusing on any input element

### Expected behaviour:
Creates / updates the patient detail.

### Observed behaviour:
Opens up: import from external results page.

image

image

### PR DESCRIPTION
**Pull request by Ashesh3**
Fixes #3727

TransparentGuineapig

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

File: src/Components/Patient/PatientRegister.tsx [hunk 1]
```diff
@@ -1035,17 +1036,17 @@ export const PatientRegister = (props: PatientRegisterProps) => {
...
+                <ButtonV2
+                  className="mb-8 mx-4"
+                  onClick={(_) => {
+                    setShowImport(true);
+                    setQuery({ extId: "" }, { replace: true });
+                  }}
+                >
+                  {" "}
+                  Import From External Results
+                </ButtonV2>
```
Comment (rithviknishad) [inline src/Components/Patient/PatientRegister.tsx]: How about also adding an import icon?

```suggestion
                <ButtonV2
                  className="mb-8 mx-4 flex gap-2 items-center"
                  onClick={(_) => {
                    setShowImport(true);
                    setQuery({ extId: "" }, { replace: true });
                  }}
                >
                  <CareIcon className="care-l-import text-lg" />
                  Import From External Results
                </ButtonV2>
```

Comment (Ashesh3) [reply src/Components/Patient/PatientRegister.tsx]: Looks good! Nice one.

**Comment by nihal467** [comment]
LGTM

**Comment by khavinshankar** [comment]
@Ashesh3 can you merge this with the develop branch and after merging if cypress fails, please do resolve that

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**